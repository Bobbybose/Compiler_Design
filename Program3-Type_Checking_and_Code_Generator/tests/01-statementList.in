int a = 2; float c = 3.1;
a = 1;