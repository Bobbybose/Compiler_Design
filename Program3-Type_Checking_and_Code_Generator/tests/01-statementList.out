Code:
%t1 = 2
a = %t1
%t2 = 3.100000
c = %t2
%t3 = 1
a = %t3

